src/iop/flip.c

```diff
@@ -95,6 +95,15 @@ int default_colorspace(dt_iop_module_t *self, dt_dev_pixelpipe_t *pipe, dt_dev_p
   return iop_cs_rgb;
 }
 
+const char *description(struct dt_iop_module_t *self)
+{
+  return dt_iop_set_description(self, _("rotate the image 90 degrees at a time."),
+                                      _("corrective"),
```

Comment:
"corrective and creative"

I sometime rotate image as a creative step, mostly for abstract images.